pkg/syslwrapper/app_test.go

```diff
@@ -0,0 +1,448 @@
+//nolint:lll
+package syslwrapper
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/anz-bank/sysl/pkg/parse"
+	"github.com/anz-bank/sysl/pkg/sysl"
+	"github.com/anz-bank/sysl/pkg/syslutil"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func readSyslModule(filename string) (*sysl.Module, error) {
```

Comment:
this function is the same as ` parse.NewParser().Parse(filename, afero.NewOsFs())`

Reply:
Cheers, will replace that!